types/msg.go

```diff
@@ -248,6 +248,7 @@ type RedelegateMsg struct {
 type DistributionMsg struct {
 	SetWithdrawAddress      *SetWithdrawAddressMsg      `json:"set_withdraw_address,omitempty"`
 	WithdrawDelegatorReward *WithdrawDelegatorRewardMsg `json:"withdraw_delegator_reward,omitempty"`
+	FundCommunityPool       *FundCommunityPoolMsg       `json:"fund_community_pool",omitempty"`
```

Comment:
This tag is not correctly formatted. A linter found this before the 1.3.0 release. Fixed in https://github.com/CosmWasm/wasmvm/pull/444